package.json

```diff
@@ -56,5 +56,8 @@
     "lint-staged": "^13.2.0",
```

Comment:
This should also be pinned to `^13.2.2`: https://github.com/okonet/lint-staged/releases/tag/v13.2.2

Reply:
Might be something to consider. My objective was the path of least resistance but this is probably the way.